ingest/mappings/mineral-observation.json

```diff
@@ -148,6 +142,25 @@
             "type":"string"
           }
         }
+      },
+      "mineral-specimen":{
```

Comment:
:+1: 

will information on how the age was determined (G, B, Y, O, R) be encoded in a property of the specimen?